design-system/react/src/components/InputAmount/InputAmount.tsx

```diff
@@ -32,19 +35,22 @@ type InputAmountComponent = FC<InputAmountProps> & {
 export const InputAmount: InputAmountComponent = ({
   name,
   balance,
+  balanceUnits = 3,
```

Comment:
should it be `balancePrecision` ? units and precision are different.

Reply:
you're right